approval_polls/tests.py

```diff
@@ -8,14 +8,16 @@
 
 from approval_polls.models import Poll
 
-def create_poll(question, days=0, ballots=0):
+def create_poll(question, username, days=0, ballots=0):
```

Comment:
This will work with `username='user1'` in the `create_poll` method
```
create_poll(question="Past poll.", days=-30)
create_poll(question="Future poll.", username="user2", days=30)
```
The first call will create a poll with username 'user1' and the second call will create a poll with username 'user2'. 'user1' is just a default value, you can overwrite it when you call the method.